src/Homepage/index.js

```diff
@@ -50,7 +53,8 @@ class Homepage extends React.Component {
     translations: PropTypes.object.isRequired,
     ipLocation: PropTypes.object,
     geoLocation: PropTypes.object,
-    setIPlocation: PropTypes.func.isRequired
+    setIPlocation: PropTypes.func.isRequired,
+    location: PropTypes.object.isRequired,
```

Comment:
this is the router location, right? any change we could rename it to routerLocation? or maybe just add a comment?